src/components/components/schedule/Event.js

```diff
@@ -17,8 +17,12 @@ class Event extends React.Component {
                 <div className="day-img">
                 </div>
                 <div className="day-info">
```

Comment:
This tag along with the div wrapping time and title individually won't be needed if you make the changes written in the next comment, but it is up to what changes you want to make, so adjust as you see fit! Some div tags may not always be necessary.